README.rst

```diff
@@ -183,6 +159,12 @@ Installing the Driver on Linux and macOS
 #. Copy :code:`pdo_snowflake.so` from the directory where you built the driver to the PHP extension directory (the same directory
    that contains the :code:`pdo.so` file).
 
+   To find the PHP extension directory, run:
+
+   .. code-block:: bash
+
+       $PHP_HOME/bin/php -i | grep '^extension_dir'
+
 #. Copy :code:`cacert.pem` from the :code:`libsnowflakeclient` subdirectory in the repository to the directory containing the
    PHP configuration files (e.g. :code:`/etc/php/7.2/fpm/conf.d` for PHP-FPM version 7.2 on Ubuntu).
```

Comment:
Is it worth mentioning that if users want to test this through the CLI, they should add the configuration files to `/etc/php/7.2/cli/conf.d/`?